packages/app/src/vis-packs/core/heatmap/MappedHeatmapVis.tsx

```diff
@@ -99,8 +124,8 @@ function MappedHeatmapVis(props: Props) {
       <HeatmapVis
         className={visualizerStyles.vis}
         dataArray={dataArray}
-        title={title}
-        dtype={formatNumLikeType(dataset.type)}
+        title={`${title}${isComplex ? ` (${COMPLEX_VIS_TYPE_LABELS[complexVisType].toLowerCase()})` : ''}`}
+        dtype={!isComplex ? formatNumLikeType(type) : undefined}
```

Comment:
The tooltip could probably be improved for complex datasets, but I chose to replicate the current behaviour for now.